pkgs/os-specific/linux/iomelt/default.nix

```diff
@@ -17,11 +21,13 @@ in stdenv.mkDerivation {
       --replace /usr $out
   '';
 
+  nativeBuildInputs = [ makeBinaryWrapper ];
```

Comment:
this looks unused

Reply:
without this there is no `iomelt` executable in `nixpkgs-review`

Reply:
can you double check ? I commented this line in your pr, rebuilt, and got:
```
result
├── bin
│   ├── iomelt -> iomelt-0.7
│   └── iomelt-0.7
└── share
    └── man
        └── man1
            └── iomelt.1.gz
```

Reply:
you were right, i must missed something, thanks for pointint it out